snap/snapcraft.yaml

```diff
@@ -326,6 +326,13 @@ apps:
       SecretService_TokenPath: $SNAP_DATA/secrets/edgex-security-proxy-setup/secrets-token.json
       SecretService_CACertPath: $SNAP_DATA/secrets/ca/ca.pem
     plugs: [home, removable-media, network]
+  secrets-config:
+    adapter: none
+    # not a mistake--secrets-config re-uses security-proxy-setup's configuration.toml
+    command: bin/secrets-config -confdir $SNAP_DATA/config/secrets-proxy-setup/res
```

Comment:
The dir is actually `$SNAP_DATA/config/security-proxy-config/res`.

Reply:
Fixed the path.